docker/build/assets.Dockerfile

```diff
@@ -154,30 +216,91 @@ RUN echo "Patching up wheel using auditwheel..." && \
         --exclude libcutensor.so.2 \
         --exclude libcutensornet.so.2 \
         --exclude libcustatevec.so.1 \
-        --exclude libcudart.so.11.0 
+        --exclude libcudart.so.11.0 \
+        --exclude libnvToolsExt.so.1
     ## [<CUDAQuantumWheel]
 
-## [Tests]
-FROM cpp_build
+# Validate that the nvidia backend was built.
+RUN if [ -z "$(ls /cuda-quantum/_skbuild/targets/nvidia.config)" ]; then \
+        echo -e "\e[01;31mError: Missing nvidia backend.\e[0m" >&2; \
```

Comment:
Do we need the ANSI color escape sequences?

Reply:
What's the issue with the color? I find it much easier to see in the large log when it is colorized. The error isn't necessarily at the bottom of the log for multi-stage builds...

Reply:
It looks like line noise if output is piped to a file for grep. But I don't have a strong objection.